frontend/src/shared/icons/Delete.tsx

```
@@ -0,0 +1,24 @@
+export const DeleteIcon = (props: SVGProps<SVGSVGElement>) => {
+  return (
+    <svg
+      xmlns='http://www.w3.org/2000/svg'
+      width='12'
+      height='12'
+      fill='none'
+      viewBox='0 0 12 12'
+      {...props}
+    >
+      <path
+        stroke='#BDBDBD'
```

Biome (2.3.14)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)